keras_cv/bounding_box/converters_test.py

```diff
@@ -14,11 +14,14 @@
 
 import itertools
 
+import numpy as np
 import tensorflow as tf
 from absl.testing import parameterized
 
 from keras_cv import bounding_box
 
+# TODO: How we want to refactor this test data to support both ragged
```

Comment:
> # TODO: How we want to refactor this test data to support both ragged

We can just split into two tests if that is easiest; no strong opinion on how this test is structured.